ui/react-components/components/CalendarPicker/CalendarPicker.jsx

```diff
@@ -14,12 +14,19 @@ import classNames from "classnames";
 
 const CalendarPicker = (props) => {
 
-    const {date, onChange, isDisabled} = props;
+    const {date, onChange, isDisabled, startDate} = props;
 
     let styles = [appointmentDatePicker];
     date ? styles.push(appointmentDatePickerSelected)
         : styles.push(appointmentDatePickerNotSelected);
-    const calendar = (<Calendar showToday={false} className={classNames(styles)}/>);
+
+    const disablePastDates = (current) => {
```

Comment:
how this "disablePastDates" different from "disablePastDates " from DatePicker.jsx file ??

Reply:
done